controller/k8s/endpoints.go

```diff
@@ -131,7 +133,7 @@ func (e *EndpointsWatcher) GetService(service string) (*v1.Service, bool, error)
 // Watches a Kubernetes resource type
 type informer struct {
 	informer cache.Controller
-	store    *cache.Store
+	store    cache.Store
```

Comment:
This field was a pointer to an interface, which meant that we had to do some funky dereferencing to access it (e.g. `(*e.serviceInformer.store).GetByKey`). Have switched it to just an interface, no pointer.